src/openforms/submissions/forms.py

```diff
@@ -0,0 +1,40 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+from .models import Submission
+
+
+class SearchSubmissionForCosignForm(forms.Form):
+    code = forms.CharField(
+        label=_("Code"),
+        help_text=_(
+            "You should have received this code in the email requesting you to cosign a form."
+        ),
+        required=True,
+        widget=forms.TextInput(
+            attrs={
+                "class": "openforms-input",
+            }
+        ),
+    )
+    form_slug = forms.CharField(
+        label=_("form slug"),
+        help_text=_("The slug of the form to which the submission corresponds."),
+        required=True,
+        widget=forms.HiddenInput(),
+        disabled=True,
```

Comment:
doesn't `disabled` result in the slug not actually being submitted in the `POST` request?

The `clean` method also doesn't use this data, but instead uses the `self.initial["form_slug"]` which doesn't seem 100% correct either.